light_curves/ML_AGNzoo.md

```diff
@@ -786,21 +778,16 @@ plt.legend(fontsize=12)
 plt.axis('off')
 ```
 
-## About this Notebook
-This notebook is created by the IPAC science platform team as a usecase of ML for time domain astrophysics. For questions contact: shemmati@caltech.edu
-
-**Author:** Shoubaneh Hemmati, Research scientist
-
-**Updated On:** 2025-03-31
+## About this notebook
 
+- **Authors:** Shoubaneh Hemmati (IRSA Research Scientist) and the Fornax team
```

Comment:
somewhere maybe we should have a `fornax team` page? (definitely not on the notebook website though)

Reply:
That seems like a good idea but I'm not sure how to implement it in practice. Things like who all should be included and how do we keep it updated while still giving credit to people who leave for work they did while here.

Reply:
(we could definitely have a notebooks only docs, this could live there)